test/fixtures/parser/postgres/postgres_comment_on.sql

```diff
@@ -0,0 +1,46 @@
+COMMENT ON TABLE mytable IS 'This is my table.';
+COMMENT ON TABLE mytable IS NULL;
+--COMMENT ON ACCESS METHOD gin IS 'GIN index access method';
+--COMMENT ON AGGREGATE my_aggregate (double precision) IS 'Computes sample variance';
+--COMMENT ON CAST (text AS int4) IS 'Allow casts from text to int4';
```

Comment:
Depending on decision above on going more generic, it may be possible to implement some or all of these, even if the underlying object itself is not supported. One one hand that may seem a bit pointless, on the other hand it would avoid issues if someone implements the object but forgets to implement the `COMMENT ON` syntax as part of that.

Reply:
Reviewed these and most look to be non-generic. Let's leave as is. Resolving.